content/docs/resiliency/usecases.md

```diff
@@ -38,3 +38,5 @@ CSM for Resiliency's design is focused on detecting the following types of hardw
 3. Array I/O Network failure is detected by polling the array to determine if the array has a healthy connection to the node. The capabilities to do this vary greatly by array and communication protocol type (Fibre Channel, iSCSI, NFS, NVMe, or PowerFlex SDC IP protocol). By monitoring the Array I/O Network separately from the Control Plane Network, CSM for Resiliency has two different indicators of whether the node is healthy or not.
 
 4. K8S Control Plane Failure. Control Plane Failure is defined as failure of kubelet in a given node. K8S Control Plane failures are generally discovered by receipt of a Node event with a NoSchedule or NoExecute taint, or detection of such a taint when retrieving the Node via the K8S API.
+
+5. CSI Driver node pods. CSM for Resiliency monitors CSI driver node pods and for some reason if CSI Driver node pods fail and enter not Ready state, it will taint the node with NoSchedule value. This will disable kubernetes scheduler to schedule new workloads on the given node, hence avoid workloads that needed CSI Driver pods to be in Ready state.
```

Comment:
Insert a period after pods. Change "for some reason if" to "If for any reason the". Insert "do not before "enter". Insert "the" before "not Ready state"

Reply:
addressed